package.json

```diff
@@ -34,88 +34,89 @@
     "emojione-picker": "^2.2.1",
     "es6-symbol": "^3.1.1",
     "escape-html": "^1.0.3",
-    "express": "^4.14.1",
+    "express": "^4.15.2",
     "extract-text-webpack-plugin": "^2.1.0",
     "file-loader": "^0.11.1",
     "font-awesome": "^4.7.0",
     "glob": "^7.1.1",
+    "history": "^3.3.0",
     "http-link-header": "^0.8.0",
     "immutable": "^3.8.1",
     "intl": "^1.2.5",
     "is-nan": "^1.2.1",
-    "js-yaml": "^3.8.3",
+    "js-yaml": "^3.8.4",
     "lodash": "^4.17.4",
     "node-sass": "^4.5.2",
-    "npmlog": "^4.0.2",
+    "npmlog": "^4.1.0",
     "object-assign": "^4.1.1",
     "path-complete-extname": "^0.1.0",
-    "pg": "^6.1.2",
-    "postcss-loader": "^1.3.3",
-    "postcss-smart-import": "^0.6.12",
+    "pg": "^6.1.5",
+    "postcss-loader": "^2.0.5",
+    "postcss-smart-import": "^0.7.0",
     "precss": "^1.4.0",
-    "prop-types": "^15.5.8",
+    "prop-types": "^15.5.10",
     "punycode": "^2.1.0",
     "rails-erb-loader": "^5.0.0",
-    "rails-ujs": "^5.1.0",
+    "rails-ujs": "^5.1.1",
     "react": "^15.5.4",
     "react-addons-perf": "^15.4.2",
     "react-addons-shallow-compare": "^15.5.2",
-    "react-autosuggest": "^7.0.1",
+    "react-autosuggest": "^9.0.1",
     "react-dom": "^15.5.4",
     "react-imageloader": "^2.1.0",
     "react-immutable-proptypes": "^2.1.0",
     "react-immutable-pure-component": "^0.0.4",
-    "react-intl": "^2.1.5",
-    "react-motion": "^0.4.5",
-    "react-notification": "^6.6.0",
+    "react-intl": "^2.3.0",
+    "react-motion": "^0.5.0",
+    "react-notification": "^6.7.0",
     "react-redux": "^5.0.4",
-    "react-redux-loading-bar": "2.4.1",
-    "react-router": "^2.8.0",
-    "react-router-scroll": "^0.3.2",
-    "react-simple-dropdown": "^1.1.4",
-    "react-toggle": "^2.1.1",
-    "redis": "^2.6.5",
+    "react-redux-loading-bar": "^2.9.0",
+    "react-router": "^3.0.5",
+    "react-router-scroll": "^0.4.2",
+    "react-simple-dropdown": "^3.0.0",
+    "react-toggle": "^3.0.1",
+    "redis": "^2.7.1",
     "redux": "^3.6.0",
-    "redux-immutable": "^3.1.0",
+    "redux-immutable": "^4.0.0",
     "redux-thunk": "^2.2.0",
-    "reselect": "^2.5.4",
-    "sass-loader": "^6.0.3",
-    "stringz": "^0.1.2",
-    "style-loader": "^0.16.1",
+    "reselect": "^3.0.1",
+    "sass-loader": "^6.0.5",
+    "stringz": "^0.2.0",
+    "style-loader": "^0.17.0",
     "uuid": "^3.0.1",
     "uws": "^0.14.5",
-    "webpack": "^2.4.1",
+    "webpack": "^2.5.1",
     "webpack-manifest-plugin": "^1.1.0",
     "webpack-merge": "^4.1.0",
     "websocket.js": "^0.1.7"
   },
   "devDependencies": {
     "@kadira/storybook": "^2.35.3",
-    "babel-eslint": "^7.2.2",
+    "babel-eslint": "^7.2.3",
     "chai": "^3.5.0",
     "chai-enzyme": "^0.6.1",
     "enzyme": "^2.8.2",
     "eslint": "^3.19.0",
-    "eslint-plugin-jsx-a11y": "^5.0.0",
-    "eslint-plugin-react": "^6.10.3",
-    "jsdom": "^9.11.0",
+    "eslint-plugin-jsx-a11y": "^5.0.1",
+    "eslint-plugin-react": "^7.0.1",
```

Comment:
https://github.com/yannickcr/eslint-plugin-react/releases/tag/v7.0.0

> ### Breaking
>
> Update rules for React 15.5.0:
> ...
> `jsx-wrap-multilines` now checks arrow functions without block body. It can be deactivated in [rule options](https://github.com/yannickcr/eslint-plugin-react/blob/v7.0.0/docs/rules/jsx-wrap-multilines.md#rule-details) ([#790](https://github.com/yannickcr/eslint-plugin-react/issues/790) [**@ColCh**](https://github.com/ColCh))
> ...